deploy/docker-swarm/clickhouse-setup/docker-compose.yaml

```diff
@@ -31,8 +31,9 @@ services:
   query-service:
     image: signoz/query-service:0.7.3
     command: ["-config=/root/config/prometheus.yml"]
-    ports:
-      - "8080:8080"
+    # ports:
+    #   - "6060:6060"     # pprof port
+    #   - "8080:8080"     # query-service port
```

Comment:
Why remove the query-service port?

Reply:
We don't need to expose _query-service_ port `8080` publicly as communication between with _query-service_ and _frontend_ happens within an internal network.